docs/source/traits_user_manual/defining.rst

```diff
@@ -687,7 +693,7 @@ will raise a TraitError if a value of any other type is assigned. For example::
 .. _union:
 
 Union
-::::::
+:::::
```

Comment:
Suggestion: `Either` is immediately above this in the text.  Perhaps  have `Union` come first?

Reply:
Done.